lib/config/config.go

```diff
@@ -94,12 +95,7 @@ type SharedDestinationConfig struct {
 }
 
 type SharedTransferConfig struct {
-	AdditionalDateFormats []string `yaml:"additionalDateFormats"`
-
-	// CreateAllColumnsIfAvailable - If true, we will create all columns if the metadata is available regardless of
-	// whether we have a value from the column. This will also bypass our Typing library.
-	// This only works for data sources with a schema such as Postgres and MySQL
-	CreateAllColumnsIfAvailable bool `yaml:"createAllColumnsIfAvailable"`
+	typing.Settings
```

Comment:
Woohoo!

Reply:
@nathan-artie Can you add a test to make sure the yaml gets unmarshalled correctly w/ the anonymous struct?

Reply:
Hah they don't, going to add `ToTypingSettings` function instead.